airbyte-integrations/connectors/source-bigquery/src/main/resources/spec.json

```diff
@@ -0,0 +1,32 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/source/bigquery",
+  "supportsIncremental": true,
+  "supportsNormalization": true,
+  "supportsDBT": true,
+  "supported_destination_sync_modes": ["overwrite", "append", "append_dedup"],
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "BigQuery Destination Spec",
+    "type": "object",
+    "required": ["project_id", "dataset_id"],
```

Comment:
elsewhere in the code dataset ID is not always assumed to exist, how come it's required here?